### ISSUE TITLE
GNS3 2.2.7 configure template window does not open

### ISSUE THREAD

**Issue by sliddjur** [GNS3/gns3-gui#2965]
**Describe the bug**
Right click on a template and select `configure template` does not open the configure template window. 

**GNS3 version and operating system (please complete the following information):**
 - OS: Ubuntu 19.10. GNS3 client installed from PPA.
 - GNS3 version: gns3 2.2.7
 - Any use of the GNS3 VM or remote server: *only* remote server. Local server disabled.

**To Reproduce**
When I right click on a template and select `configure template` the window does not open. However, I can configure settings on a single node that is already in the topology.

A workaround is to use open a new GNS3 window and then suddenly the `configure template` button work on new window. I am not completely sure whats triggers the error at first. but if I would have started working in that new window, the configure template button would have worked for a while, until it didnt.

**Screenshots or videos**
![gns3](https://user-images.githubusercontent.com/10002057/79119463-b8f44400-7d90-11ea-9046-7eeb68555180.gif)

**Additional context**
No

**Comment by jpobeda** [comment]
Same here. Basically same setup. Works the first time and then I get exceptions.

```

****** Exception detected, traceback information saved in /home/sor/.config/GNS3/2.2/profiles/GNS3_VM/exceptions.log ******

PLEASE REPORT ON https://www.gns3.com

AttributeError: __getattribute__

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/share/gns3/gns3-gui/lib/python3.7/site-packages/gns3/qt/__init__.py", line 257, in partial
    return func(*args, **kwargs)
  File "/usr/share/gns3/gns3-gui/lib/python3.7/site-packages/gns3/nodes_view.py", line 209, in _configurationSlot
    dialog = ConfigurationDialog(template.name(), template.settings(), configuration_page(), parent=self)
  File "/usr/share/gns3/gns3-gui/lib/python3.7/site-packages/gns3/dialogs/configuration_dialog.py", line 41, in __init__
    self.setupUi(self)
  File "/usr/share/gns3/gns3-gui/lib/python3.7/site-packages/gns3/ui/configuration_dialog_ui.py", line 64, in setupUi
    QtCore.QMetaObject.connectSlotsByName(configurationDialog)
SystemError: <built-in function connectSlotsByName> returned a result with an error set
```

**Comment by grossmj** [comment]
https://github.com/GNS3/gns3-gui/issues/2819#issuecomment-533876471

You need to upgrade to PyQt 5.13.1 or later.

**Comment by grossmj** [comment]
Refactoring ConfigurationDialog() to remove @property fixes the issue. This fix will be included in version 2.2.8.

Thanks for https://github.com/autokey/autokey/issues/313#issuecomment-546614545 to pointing out the problem.
